src/TaskCard.js

```diff
@@ -5,32 +5,32 @@ import Popup from "reactjs-popup";
 import "reactjs-popup/dist/index.css";
 import "./App.css";
 
-const TaskCard = ({ title, display, description, id }) => {
-  const [taskName, setTaskName] = useState(title);
-  const [taskDescription, setTaskDescription] = useState(description);
-  const [isOpen, setOpen] = useState(false);
-  const handleEditTask = (e) => {
+const TaskCard = ({ id,title, isComplete, description, updateStatusHandler , editHandler}) => {
```

Comment:
```suggestion
const TaskCard = ({ id, title, isComplete, description, updateStatusHandler, editHandler }) => {
```
Formatting issue, Please use Prettier for autoformatting, [how to integrate prettier in vscode, see here](https://blog.avenuecode.com/how-to-integrate-prettier-and-eslint-in-vscode-and-react)